### PR TITLE
Hotfix/bugs by form upgrade

### PR DESCRIPTION
**Pull request by hujiahao-hjh**
## What type of PR is this?

- [ ] Feature
- [x] Bugfix
- [ ] Test
- [ ] Documentation
- [ ] Refactor
- [ ] Style
- [ ] Chore

## Which issue(s) this PR fixes:

Fixes # bugs by Table upgrade and styles bugs

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [x] No

### CODE REVIEW

File: shell/app/common/components/form-modal.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ class FormModalComp extends React.Component<IProps, IState> {
             form.setFieldsValue(pureData);
           } else {
             // 通过传入的formRef来获取fieldsStore，否则setFieldsValue报错
```
Comment (daskyrk) [inline shell/app/common/components/form-modal.tsx:87]: remove this useless comment

File: shell/app/locales/en.json [hunk 2]
```diff
@@ -3644,7 +3644,8 @@
...
+    "1. Input parameter reference for this scene: / $<< params.lab >>, and lab is input parameter name of the scene; 2. Output parameter reference for the front interface: / $<< outputs.10.status >>, and 10 is the interface ID, status is the output parameter name; 3. Mock parameter reference: / $<< random.string >>, and string is a function to randomly generate a string (includes integer, float, boolean, upper, lower, etc.); 4. Global environment variable reference: / $<< configs.autotest.cluster-name >>, and cluster-name is the environment variable name.": "1. Input parameter reference for this scene: / $<< params.lab >>, and lab is input parameter name of the scene; 2. Output parameter reference for the front interface: / $<< outputs.10.status >>, and 10 is the interface ID, status is the output parameter name; 3. Mock parameter reference: / $<< random.string >>, and string is a function to randomly generate a string (includes integer, float, boolean, upper, lower, etc.); 4. Global environment variable reference: / $<< configs.autotest.cluster-name >>, and cluster-name is the environment variable name."
```
Comment (daskyrk) [inline shell/app/locales/en.json:3648]: use a short key

File: core/src/nusi/index.tsx [hunk 3]
```diff
@@ -55,7 +55,6 @@ import {
 import { FixedSelect } from './fixed-select';
 import FixRangePicker from './range-picker';
 import '@terminus/nusi/dist/nusi.scss';
-import 'antd/dist/antd.less';
 import 'antd-latest/dist/antd.less';
```
Comment (daskyrk) [inline core/src/nusi/index.tsx:58]: Remove the antd@3.x in package.json and rename 4.x to antd.

File: shell/app/styles/antd-extension.scss [hunk 4]
```diff
@@ -412,24 +405,37 @@ div.ant-tabs-nav::before,
...
   line-height: 28px;
   overflow: visible;
 }
 
 .ant-table-tbody > tr > td {
   color: $color-dark-6;
 }
+
+.pk-form-item-children .ant-select {
+  width: 100%;
```
Comment (laojun) [inline shell/app/styles/antd-extension.scss:440]: it's there still a pk-form exist

**Review by daskyrk — APPROVED** [review]

**Pull request merged**